.github/workflows/workflow.yml

```diff
@@ -13,27 +13,50 @@ jobs:
           - macos-latest
           - ubuntu-latest
           - windows-latest
+        ocaml-version:
+          - 4.06.1
```

Comment:
Why 4.06?

Reply:
I thought we needed this version for testing with BuckleScript. Maybe it's not?

Reply:
Oh yes. Although I'm not sure we actually have any bucklescript tests at the moment.